ring/lifecycler.go

```diff
@@ -25,19 +25,19 @@ import (
 
 var (
 	consulHeartbeats = promauto.NewCounterVec(prometheus.CounterOpts{
-		Name: "cortex_member_consul_heartbeats_total",
+		Name: "member_consul_heartbeats_total",
```

Comment:
These metrics are registered globally (BTW, these are the type of metrics we don't want in dskit). Removing the prefix from here, it means when we'll update dskit in Cortex, these metrics will not have the `cortex_` prefix anymore and there will be no way to add it back.